src/Topic.hs

```diff
@@ -49,33 +49,33 @@ instance Situationable Situations where
 -- The most common Situation parameters are letting anyone be vulnerable, and
 -- either letting anyone be the dealer, ensuring North is an unpassed hand, or
 -- ensuring South is an unpassed hand. Make some syntactic sugar for that.
-wrapVulDlr :: State StdGen (Vulnerability -> Direction -> Situation) ->
```

Comment:
Rename these to match! The plan had been to completely redo them, but that is on hold until later, given that the plan didn't actually work.